tests/suites/test_suite_pkparse.function

```diff
@@ -3,6 +3,17 @@
 #include "mbedtls/pem.h"
 #include "mbedtls/oid.h"
 #include "mbedtls/legacy_or_psa.h"
+
+/*
+ * When testing with "tests/scripts/depends.py pkalgs", one of the complementary
+ * domains is with !MBEDTLS_ECP_C, which cause the symbol
+ * MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE to be not defined. However this is
+ * the returned error from "mbedtls_ecp_sw_derive_y()" when trying to read
+ * a compressed point from a curve which is not supported.
+ */
+#ifndef MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE
+#define MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE               -0x4E80
+#endif
```

Comment:
I think we could achieve the same effect with `#include "mbedtls/ecp.h"` as the error codes are defined unconditionally.

Reply:
Your solution is way more elegant than mine, of course, so I just force pushed it taking the old one away ;)